lib/src/style/interactive_features/standard_buildings.dart

```diff
@@ -19,18 +27,18 @@ extension StandardBuildingsFeature
   }
 }
 
-/// A Featureset of buildings in the Standard style
+/// A featureset of StandardBuildings features.
 class StandardBuildings extends FeaturesetDescriptor {
   StandardBuildings({String importId = "basemap"})
-      : super(featuresetId: "buildings", importId: importId);
+    : super(featuresetId: "buildings", importId: importId);
 }
 
-/// Represents available states for Buildings in the Standard style.
-class StandardBuildingState extends FeatureState {
-  /// When `true`, the feature is highlighted. Use this state to create a temporary effect (e.g. hover).
+/// Represents available states for StandardBuildings features in the Standard style.
+class StandardBuildingsState extends FeatureState {
```

Comment:
Bit of a breaking change, should we mention it in changelog? Alternatively, maybe we could have a typedef to the old name for compatibility.

Reply:
I think since it is experimental we can just mention it in the changelog. I've added an update in the latest commit.